.changeset/selfish-steaks-behave.md

```diff
@@ -0,0 +1,18 @@
+---
+"@hashicorp/design-system-components": major
+"@hashicorp/design-system-tokens": major
+---
+
```

Comment:
@didoo I think it would be worth adding a note here as well about the significance of the 1.0 release so it's more clear why this release happens to be 1.0

Reply:
good idea. do you have time to do it?

Reply:
on second thought, i'll do this but _in_ the actual version PR. It'll be easier to format the markdown correctly there.